modules/data.atmosphere/R/met.process.R

```diff
@@ -293,16 +295,21 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
 
   #--------------------------------------------------------------------------------------------------#
   # Prepare for Model
+  # Determine output format name and mimetype
+  
+  model_info <- db.query(paste0("SELECT f.name, f.id, mt.type_string from modeltypes as m",
+                              " join modeltypes_formats as mf on m.id = mf.modeltype_id",
+                              " join formats as f on mf.format_id = f.id",
+                              " join mimetypes as mt on f.mimetype_id = mt.id",
+                              " where m.name = '", model, "' AND mf.tag='met'"),con)
+
+  if (model_info[1] == "CF Meteorology"){
+    stage$met2model=FALSE
+  } 
 
   if(stage$met2model == TRUE){
```

Comment:
Incidentally, `== TRUE` is redundant, and this expression is 100% equivalent to simply `if(stage$met2model)`, though I guess it is more explicit. You don't have to change this, just thought I'd point it out.